dlang.org.ddoc

```diff
@@ -552,3 +552,10 @@ PROJECT_SOURCE_DIR=
 
 FA_ICON=<i class="fa fa-$1" aria-hidden="true"></i>
 _=
+
+_ = Opt-out of automatic keyword highlighting - see https://dlang.org/changelog/2.079.0.html#fix18361
+DDOC_AUTO_PSYMBOL =
+DDOC_AUTO_KEYWORD =
+DDOC_AUTO_PARAM =
+_ = DDOC_AUTO_PSYMBOL_SUPPRESS = FIXME_UNDERSCORE_PREFIX
```

Comment:
why?

Reply:
Because DDoc doesn't support comments and `DDOC_AUTO_PSYMBOL_SUPPRESS` is within reach. After all, there are still tons of underscores used to suppress auto highlighting like `_range` references in Phobos.